CMakeLists.txt

```diff
@@ -74,6 +74,11 @@ if (USE_OPENMP)
   target_link_libraries(intgemm PUBLIC OpenMP::OpenMP_CXX)
 endif()
 
+if (COMPILE_WASM)
+    # A compile defintion to compile intgemm on WASM platform
+    target_compile_definitions(intgemm PUBLIC WASM)
```

Comment:
Please support older cmake here.

Reply:
@kpu This cmake file seems to be using a lot of new stuff at lots of places. e.g. [1](https://github.com/kpu/intgemm/blob/master/CMakeLists.txt#L61), [2](https://github.com/kpu/intgemm/blob/master/CMakeLists.txt#L64), [3 and 4](https://github.com/kpu/intgemm/blob/master/CMakeLists.txt#L88-L92).

Let me know your final call on this. I can downgrade it to support older cmake 👍

Reply:
What I can tell you is currently everything works on cmake 3.5.1 from Ubuntu 16.04.6 LTS but this doesn't.

Reply:
This is weird. The same [cmake command](https://github.com/kpu/intgemm/blob/master/CMakeLists.txt#L79) works and this doesn't?

Anyway, what should I change it to? `add_compile_definitions`?

Reply:
@kpu Ping on this one.